README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
Consider adding a brief description of what 'go-task' is and its purpose. This helps users understand its context better.